lib/mixlib/install/cli.rb

```diff
@@ -32,7 +33,9 @@ def list_versions(product_name, channel)
       option :platform_version,
         aliases: ["-l"]
       option :architecture,
-        aliases: ["-a"]
+        default: "x86_64",
+        aliases: ["-a"],
+        enum: %w{i386 s390x sparc x86_64}
```

Comment:
Nope I think you are correct @scotthain, I'm actually going to add a constant to the Options class to hold the valid archs.